Resources/public/js/views/ez-universaldiscoveryview.js

```diff
@@ -44,6 +52,71 @@ YUI.add('ez-universaldiscoveryview', function (Y) {
             });
 
             this._publishEvents();
+
+            this.on('changeTab', this._updateVisibleMethod);
+        },
+
+        /**
+         * Updates the tab to show the correct tab depending on the visible
+         * method
+         *
+         * @method _uiUpdateTab
+         * @protected
+         */
+        _uiUpdateTab: function () {
+            var container = this.get('container'),
+                htmlId = '#' + this._visibleMethodView.getHTMLIdentifier();
+
+            this._selectTab(
+                this._getTabLabel(container.one('[href="' + htmlId + '"]')),
+                htmlId,
+                container
+            );
+        },
+
+        /**
+         * Updates the visible flag of the method views depending on the value
```

Comment:
https://github.com/ezsystems/PlatformUIBundle/pull/160/files#diff-c94326efd4810afcf1f696312d405c97R306
The universal discovery view has several *methods*. A method is view which provides a way to discover content. At first, we'll implement the *browse* method which basically displays the content as a tree, later we'll provide the *recent* method to display the last visited contents and after we can provide others (search, bookmark, ...)